### PR TITLE
Add skuSelector and Share components

### PR DESCRIPTION
**Pull request by andreldsa**
#### What is the purpose of this pull request?
Integrating the `SkuSelector` and `Share` components.

#### How should this be manually tested?
Access: https://andre--storecomponents.myvtex.com/porsche-911/p

#### Screenshots or example usage
image

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [x] New feature (a non-breaking change which adds functionality)
* [x] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/ProductDetails.js [hunk 1]
```diff
@@ -66,16 +76,39 @@ class ProductDetails extends Component {
...
-                skuId={selectedItem.itemId}
-              >
-                <FormattedMessage id="button-label" />
-              </BuyButton>
+            <div className="flex w-100 pv2">
+              <div className="pv2 pr3 f6">
+                <FormattedMessage id="share-label" />:
+              </div>
+              <Share 
+                options={{'size': '25'}}
```
Comment (estacioneto) [inline react/ProductDetails.js]: Size is a number prop

File: react/ProductDetails.js [hunk 2]
```diff
@@ -66,16 +76,39 @@ class ProductDetails extends Component {
...
-              >
-                <FormattedMessage id="button-label" />
-              </BuyButton>
+            <div className="flex w-100 pv2">
+              <div className="pv2 pr3 f6">
+                <FormattedMessage id="share-label" />:
+              </div>
+              <Share 
+                options={{'size': '25'}} 
+                social={{'Facebook': true, 'WhatsApp': true, 'Twitter': true }}
```
Comment (estacioneto) [inline react/ProductDetails.js]: These are the default values. Will you pass them anyway?

**Review by estacioneto — CHANGES_REQUESTED** [review]

File: react/ProductDetails.js (hunk 2 above)
Comment (andreldsa) [reply react/ProductDetails.js]: Nops.

File: react/ProductDetails.js (hunk 1 above)
Comment (andreldsa) [reply react/ProductDetails.js]: Nice.

**Review by estacioneto — APPROVED** [review]

**Review by oldufre — APPROVED** [review]

File: react/ProductDetails.js [hunk 3]
```diff
@@ -8,6 +8,8 @@ import ProductName from 'vtex.store-components/ProductName'
 import Price from 'vtex.store-components/ProductPrice'
 import ProductImages from 'vtex.store-components/ProductImages'
 import ShippingSimulator from 'vtex.store-components/ShippingSimulator'
+import SKUSelector from 'vtex.store-components/SKUSelector'
```
Comment (brunojdo) [inline react/ProductDetails.js]: `import { BuyButton, ProductDescription, ProductName, ProductPrice, ProductImages, ShippingSimulator, SKUSelector, Share } from 'vtex.store-components'
`

**Review by brunojdo — CHANGES_REQUESTED** [review]

**Review by brunojdo — APPROVED** [review]

**Pull request merged**